grails-app/assets/javascripts/profileEditor/controllers/UserAccessController.js

```diff
@@ -80,10 +80,22 @@ profileEditor.controller('UserAccessController', function (messageService, util,
         });
     };
 
-    self.privateModeChanged = function() {
+    self.privateModeChanged = function(form) {
```

Comment:
This function is also called when opus is loaded. The user filtering and subsequent update should only run when user has clicked private collection button and not when opus is loaded.